.github/workflows/artifacts-upload/action.yml

```diff
@@ -2,52 +2,74 @@ name: 'Upload elements build artifacts'
 description: 'Upload elements build artifacts that can be used by release workflows'
 author: 'Benjamin Pagelsdorf'
 
+inputs:
+  upload-core-artifacts:
+    description: 'Do upload core artifacts?'
+    required: false
+    default: 'true'
+  upload-storybook-artifact:
+    description: 'Do upload storybook artifact?'
+    required: false
+    default: 'true'
+  upload-example-artifacts:
+    description: 'Do upload example project artifacts?'
+    required: false
+    default: 'true'
```

Comment:
IMO the default values should be set to `false`. That way you have to explicitly set these values which make for better readability. 

e.g.:
This is better readable:
```
    - uses: ./.github/workflows/artifacts-upload
      with:
	upload-storybook-artifact: true
```
than this:
```
    - uses: ./.github/workflows/artifacts-upload
      with:
        upload-core-artifacts: false
        upload-example-artifacts: false
```